### DISCUSSION TITLE
AWS MSK ACL Creation/Modification

### DISCUSSION THREAD

**Discussion by damvinod** [provectus/kafka-ui#4487] [Q&A]
Hi, Is there an option to Create/Update ACL's from Kafka-UI. Is it not supported? I have configured Kafka-UI to use MSK IAM authentication.

<img width="1728" alt="Screenshot 2024-10-12 at 4 42 56 PM" src="https://github.com/user-attachments/assets/ebd5a202-9a9d-4b31-9be5-ba40ff9f086e">

**Comment by Haarolean** [comment]
Hi, this repo is not maintained (#4255). Please take a look at https://github.com/kafbat/kafka-ui (more on "why" [here](https://github.com/kafbat/kafka-ui/discussions/23)), release 1.0 supports editing ACLs.
